Fhi.HelseId.Web/Handlers/NoAuthenticationHandler.cs

```diff
@@ -16,8 +16,10 @@ public class NoAuthenticationHandler : AuthenticationHandler<AuthenticationSchem
     {
         private readonly NoAuthenticationUser _noAuthenticationUserConfig;
 
+#pragma warning disable CS0618 // TODO: Use TimeProvider instead. Can be removed when support for Net6.0 is removed.
         public NoAuthenticationHandler(IOptionsMonitor<AuthenticationSchemeOptions> options, ILoggerFactory logger, UrlEncoder encoder, ISystemClock clock, IOptions<HelseIdWebKonfigurasjon> helseIdWebOptions)
```

Comment:
Kan du ikke legge inn #ifdef NET6 f.eks og ha en for .net6 og en for .net 8.  Når vi da fjerner net6 så er det veldig tydelig hva som skal  fjernes

Reply:
@OsirisTerje Har implementert forslaget ditt.